.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Continuous Integration'
+on: [ push, pull_request ]
+
+env:
+  OSSRH_USERNAME: ${{ secrets.SONATYPE_S01_USERNAME }}
+  OSSRH_TOKEN: ${{ secrets.SONATYPE_S01_PASSWORD }}
+  MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_CENTRAL_GPG_PASSWORD }}
+jobs:
+  BuildAndPublish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v3
+        with: # running setup-java again overwrites the settings.xml
+          java-version: 8
```

Comment:
So, this is java 8, but the poms specify java 7, which is deprecated.

I'd say this is "better", but we need to watch for consistency:
        <maven.compiler.source>1.7</maven.compiler.source>
        <maven.compiler.target>1.7</maven.compiler.target>

Reply:
Indeed, also Java 1.8 is the lowest available Java version in GHA.